core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -2740,7 +2740,7 @@ struct ViewValueFunctor;
 
 template <class ExecSpace, class ValueType>
 struct ViewValueFunctor<ExecSpace, ValueType, false /* is_scalar */> {
-  typedef Kokkos::RangePolicy<ExecSpace> PolicyType;
+  typedef Kokkos::RangePolicy<ExecSpace, Kokkos::IndexType<int64_t>> PolicyType;
```

Comment:
This is one of the two changes which matter

Reply:
`?w=1` at the end of the url in your browser does the trick ;)

Reply:
@rrdrake @micahahoward @vbrunini It might be worth patching your Trilinos with these changes.